vscode-ext/src/tests/e2e.ts

```diff
@@ -78,7 +78,7 @@ describe("mirrord sample flow test", function () {
         await setBreakPoint(fileName, browser, defaultTimeout);
         await startDebugging();
 
-        const inputBox = await InputBox.create();
+        const inputBox = await InputBox.create(10000);
```

Comment:
What's going on here?

Reply:
we needed to extend the timeout here to 10000 since their default timeout was way too less and lead to some flakes